components/Navbar/Tab.tsx

```diff
@@ -55,6 +55,7 @@ export default function Tab({
             'hover:bg-black/25 hover:text-white hover:text-opacity-100':
               status && status.unlocked && !isActive,
             'bg-black/25 text-opacity-100': isActive,
+            'border-r': isLast,
```

Comment:
Hey @shaavan, it seems like the little `isLast` variable may have caused a bit of trouble. It looks like it wasn't declared in the `components/Navbar/Tab.tsx` file. I'm surprised that Vercel wasn't able to catch this error, but I was able to spot it when I pulled the master branch locally. 

![Screenshot 2023-02-08 at 1 59 58 PM](https://user-images.githubusercontent.com/54861487/217480311-9504587b-b579-4757-aab0-244793477bfa.png)

I think this error occured because the `isLast` Variable was removed in this [PR](https://github.com/saving-satoshi/saving-satoshi/commit/93735aa33f68bd03ef6824a77fc993929c52b8fa#diff-474acce4ba9be67e994c8743fffbf34bc4f9bd3c3ec168b851a9fededc198572L38) while your Local branch was still having the `isLast` variable